tests/test_validators.py

```diff
@@ -63,6 +66,24 @@
 ]
 
 
+VALID_SCHEMAS_FOR_MULTIPLE_OF = [
```

Comment:
you dont need anything so complex here because your validator only tests numbers not schemas

```python
@pytest.mark.parametrize("value,factor", [
    (-8.2, 0.02),
    (2.05, 0.05),
    (3.03, 1.01)
])
def test_doesnt_raises_error(value, factor):
    validator = MultipleOfValidator(factor)
    validator(value)
    assert True
```

would be fine